src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/wwwroot/theme-2019/style.css

```diff
@@ -2064,8 +2064,8 @@ footer{
 	background-size: cover;
 	background-repeat: no-repeat;
 	overflow: hidden;
```

Comment:
This chunk of code is related only to devices with screen size over 992px. For resolution between 768px and 992px padding changes again and for lower res site is showed in "mobile device mode" where Santa is is far beyond screen. For more dynamic site we might need someone more skilled in css magic :)